commands.py

```diff
@@ -496,13 +456,23 @@ async def latex(self, ctx):
         `!latex \\frac{a}{b}` [img]
         """
 
-        formula = ctx.message.content[len(self.bot.command_prefix) + 6:]
-        formula = formula.replace("%", "%25").replace("&", "%26")
-        body = "formula=" + formula + \
-            "&fsize=30px&fcolor=FFFFFF&mode=0&out=1&remhost=quicklatex.com&preamble=\\usepackage{amsmath}\\usepackage{amsfonts}\\usepackage{amssymb}&rnd=" + str(random.random() * 100)
+        formula = " ".join(*args).strip("`")
```

Comment:
args is already a list here so I don't think there's a need to spread

and i also think this will TypeError because join only takes one argument